package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-material-icon",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This will be bumped as soon as package will be republished to NPM

Reply:
Thank you :-) Didn't know that